### PR TITLE
Product.development

### PR DESCRIPTION
**Pull request by Dharmiik-Chauhan**
image upload with use of paperclip in product model and give font awsome in show edit abd distroy

### CODE REVIEW

File: app/models/product.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 class Product < ApplicationRecord
+  has_attached_file :image, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: 'user2.jpg'
```
Comment (ManojParmar-MTAJ) [inline app/models/product.rb]: use proper image name for default image like missing or image and use it here

File: app/views/products/_form.html.erb [hunk 2]
```diff
@@ -1,74 +1,93 @@
...
+      <% end %>
+    </ul>
+  </div>
 <% end %>
 <table class="table table stried" border="2" align="center" cellspacing="4" cellpadding="4">
   <tr>
     <div class="field">
       <td><%= form.label :product_name %></td>
-      <td> <%= form.text_field :product_name, id: :product_name %></td>
+      <td>
```
Comment (ManojParmar-MTAJ) [inline app/views/products/_form.html.erb:17]: remove :id attribute from all fields you don't need this

File: app/views/suppliers/index.html.erb [hunk 3]
```diff
@@ -26,9 +26,15 @@
...
-        <td><%= link_to 'Show',supplier %></td>
-        <td><%= link_to 'Edit', edit_supplier_path(supplier) %></td>
-        <td><%= link_to 'Destroy', supplier, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <td><%= link_to supplier do %>
+        <i class="fa fa-eye"></i>
+          <% end %></td>
+        <td><%= link_to edit_supplier_path(supplier) do %>
+              <i class="fa fa-pencil"></i>
+            <% end %></td>
+            <td><%= link_to  (supplier), method: :delete, data: { confirm: 'Are you sure?' } do%>
```
Comment (ManojParmar-MTAJ) [inline app/views/suppliers/index.html.erb:35]: give proper image before deleting product like 'are you sure you want to delete this product?'

**Review by ManojParmar-MTAJ — CHANGES_REQUESTED** [review]

File: app/controllers/products_controller.rb [hunk 4]
```diff
@@ -9,36 +9,34 @@ def new
...
-  end
+  def show; end
 
-  def edit
-  end
+  def edit; end
 
   def create
     @product = Product.new(product_params)
-    respond_to do|format|
```
Comment (RaviMariya-MTAJ) [inline app/controllers/products_controller.rb:20]: remove extra code of respond to
write like
if @product.save
    redirect_to @product, notice: 'product was successfully created'
else
   render :new
end

**Pull request merged**